content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -63,7 +71,32 @@ MTM-46640</td>
 
 <tr>
 <td>
- MongoDB </td>
+Core platform</td>
+<td>The identifier for managed objects is now a random number and no longer monotonically increasing.</td>
```

Comment:
"monotonically increasing" - does this mean "consecutively/serially numbered"?

Reply:
I read through the ticket and this seemed to be a known term for those that are familiar with the topic. I will ask again:-).